Gruntfile.js

```diff
@@ -626,7 +626,7 @@ module.exports = function (grunt) {
                     // ['Nightmare'] is a good alt for a current headless
                     // FIXME: fix this, PhantomJS is deprecated
                     isMac ?
```

Comment:
Ill squash this out, its just a tad annoying to see the output of all the tests twice while working on them

Reply:
Did you mean to leave this? (I'm actually OK taking it out, honestly, since I find it annoying, too. I don't think it gives much value, at least not for dev.)

Reply:
I'll leave this, the comments above clearly explain how to use `--browsers=Chrome,Firefox` to get multiple.

The coverage block is even more annoying, when you are trying to write tests, but I'm not sure how to fix that yet (keep it to annoy us, just not so much).